src/content/ui-api/kubectl-gs/template-app.md

```diff
@@ -47,41 +48,30 @@ produces the following output:
 apiVersion: application.giantswarm.io/v1alpha1
 kind: App
 metadata:
-  creationTimestamp: null
-  labels:
-    app-operator.giantswarm.io/version: 1.0.0
-  name: my-app
-  namespace: default
+  name: keda
+  namespace: 2hr7z
 spec:
-  catalog: pipo-catalog
-  config:
-    configMap:
-      name: 2hr7z-cluster-values
-      namespace: 2hr7z
-    secret:
-      name: ""
-      namespace: ""
+  catalog: giantswarm-playground
   kubeConfig:
-    context:
-      name: 2hr7z-kubeconfig
     inCluster: false
-    secret:
-      name: 2hr7z-kubeconfig
-      namespace: 2hr7z
-  name: my-app
-  namespace: 2hr7z
-  userConfig:
-    configMap:
-      name: ""
-      namespace: ""
-    secret:
-      name: ""
-      namespace: ""
+  name: keda
+  namespace: default
   version: 0.1.0
-status:
-  appVersion: ""
-  release:
-    lastDeployed: "0001-01-01T00:00:00Z"
-    status: ""
-  version: ""
 ```
+
+It also supports the following optional flags:
+
+- `--defaulting-enabled`: Don't template fields that will be defaulted. (default true)
```

Comment:
Instead of a negative phrase like "Don't template fields" I'd rather explain what this does in a positive way. E. g. "Only include details that differ from default value. When false, a much longer template is created."

Reply:
Further down I read

> For older releases you can set the `--defaulting-enabled` flag to false.

Is this flag effective for all releases, or just for some?

Reply:
Thanks, that's much better.